apps/frontend/src/components/global/Sidebar.vue

```diff
@@ -185,8 +185,7 @@ import Vue from 'vue';
 import Component from 'vue-class-component';
 import {EvaluationFile, ProfileFile} from '@/store/report_intake';
 import {InspecDataModule} from '@/store/data_store';
-import FileItem from '@/components/global/sidebaritems/SidebarFile.vue';
-import ProfileItem from '@/components/global/sidebaritems/SidebarProfile.vue';
+import FileLister from '@/components/global/sidebaritems/SidebarFileLister.vue';
```

Comment:
Preferably call this `FileList` and `SidebarFileList` in order to match the naming of the rest of the components.